### PR TITLE
Issue66

### PR DESCRIPTION
**Pull request by louisdelanoeye**
Users can now navigate with the arrows through the search list.

### CODE REVIEW

File: assets/js/search.js [hunk 1]
```diff
@@ -98,8 +124,39 @@ window.searchHandler = function() {
...
+          }
+        }
+      }else{
+        sethovering($(".search--results li").eq(0).find("a"));
+      }
+    }else if(e.keyCode == 38){
+      if($("li a.hover").length>0){
+        e.preventDefault();
+        var next =($("li a.hover").parent().prev("li"));
+        console.log(next);
```
Comment (ddmkr) [inline assets/js/search.js]: remove

**Comment by ddmkr** [comment]
Awesome

File: assets/js/search.js [hunk 2]
```diff
@@ -83,8 +84,32 @@ window.searchHandler = function() {
 
   }
 
-  searchInput.keyup(debounce(function(){
+  function hover(){
+    //var clas = document.getElementsByClassName("search--results");
```
Comment (ludov04) [inline assets/js/search.js]: remove comment

**Comment by ludov04** [comment]
Really cool :).
You can merge it when the commented code has been removed

**Pull request merged**